src/shared/components/pointer-box/PointerBox.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import React, { ReactNode, useRef, useEffect, useState } from 'react';
+import { ReactNode, useRef, useEffect, useState } from 'react';
```

Comment:
Shouldn't  `ReactNode` be a type import?

Reply:
I am sure this was the automated script 😄